examples/README.md

```diff
@@ -157,13 +157,15 @@ For this example the following is needed in the selected region:
 * Compute Engine API: Persistent Disk SSD (GB): **~50 GB**
 * Compute Engine API: Persistent Disk Standard (GB): **~50 GB static + 50
   GB/node** up to 1,250 GB
-* Compute Engine API: N2 CPUs: **12**
-* Compute Engine API: C2 CPUs: **4** for controller node and **60/node** active
-  in `compute` partition up to 1,204
+* Compute Engine API: N2 CPUs: **2** for the login node and **2/node** active 
+  in the `debug` partition up to 12
+* Compute Engine API: C2 CPUs: **4** for the controller node and **60/node** 
+  active in the `compute` partition up to 1,204
+* Compute Engine API: H3 CPUs: **88/node** active in the `h3` partition up to 1760
```

Comment:
If user goes on CloudConsole UI  "All Quotas" page - the main avenue to modify/request quota,
they can put `"N2 CPUs"` or `"C2 CPUs"` into search bar to find relevant items to edit.
`"H3 CPUs"` doesn't yield the same result.
Please add an explanation on how to find H3 quota items, options:
* put `vm_family:H3` into search bar - I think this one is better, your call.
* put `"CPUs per VM family"` into search bar